gef-ui/component-tests/partials/amendments/cancel-portal-facility-amendment.component-test.ts

```diff
@@ -0,0 +1,44 @@
+import { CancelAmendmentViewModel } from '../../../server/types/view-models/amendments/cancel-amendment-view-model';
+import pageRenderer from '../../pageRenderer';
+
+const page = 'partials/amendments/cancel.njk';
+const render = pageRenderer(page);
+
+describe(page, () => {
+  const exporterName = 'exporterName';
+  const previousPage = 'previousPage';
+  const cancelAmendmentUrl = 'cancelAmendmentUrl';
+
+  const params: CancelAmendmentViewModel = {
+    exporterName,
+    cancelAmendmentUrl,
+    previousPage,
+  };
+
+  it(`should render the 'Back' link`, () => {
+    const wrapper = render(params);
+
+    wrapper.expectLink('[data-cy="back-link"]').toLinkTo(previousPage, 'Back');
+  });
+
+  it('should render the `Yes, cancel` button', () => {
+    const wrapper = render(params);
+
+    wrapper.expectElement('[data-cy="yes-cancel-button"]').toExist();
+    wrapper.expectText('[data-cy="yes-cancel-button"]').toRead('Yes, cancel');
+    wrapper.expectPrimaryButton('[data-cy="yes-cancel-button"]').toLinkTo(undefined, 'Yes, cancel');
```

Comment:
I think this can be simplified - IIRC `expectPrimaryButton` checks both of the above
```suggestion
    wrapper.expectPrimaryButton('[data-cy="yes-cancel-button"]').toLinkTo(undefined, 'Yes, cancel');
```

Reply:
Done